lib/miq_automation_engine/engine/miq_ae_event.rb

```diff
@@ -127,5 +128,21 @@ def self.automate_user
     user
   end
 
-  private_class_method :automate_user
+  def self.call_automate(obj_type, obj_id, attrs, instance_name, 
+                         user, state = nil, message = nil)
+    args = {
+      :object_type      => obj_type,
+      :object_id        => obj_id,
+      :attrs            => attrs,
+      :instance_name    => instance_name,
+      :user_id          => user.id,
+      :miq_group_id     => user.current_group.id,
```

Comment:
we call it `:miq_group_id` everywhere else like `Rbac` - so I'd like to keep it with that.